examples/vitereactts-live/src/main.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import ReactDOM from 'react-dom/client'
+import App from './App.tsx'
+import './index.css'
```

Comment:
Please remove the index.css and App.tsx and directly use the component here.

This will reduce the amount of files we have to worry about.